app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count_masks = 0
+    for friend in friends:
```

Comment:
I would add some line breaks before and after the for-loop to make your code more readable